contents.json

```diff
@@ -27547,6 +27547,26 @@
       "license": "apache-2.0",
       "date_added": "Jan 17 2022",
       "suggested_by": "@dkhamsing"
+    },
+    {
+      "title": "My Little Popcat",
+      "category-ids": ["category-id"],
```

Comment:
Can you change this to ["misc"]

Reply:
I changed/added category id. My mistake 😅